fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java

```diff
@@ -83,11 +78,6 @@
      * Map of columns and their widths
      */
     private final Map<Integer, Double> colWidths = new HashMap<>();
-    
```

Comment:
Was there a problem with having custom row heights and repeating rows? Nevertheless don't just remove existing functionality.

Reply:
Yes, sorry, this was an accident